packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react'
+'use client'
+
+import React, { useEffect } from 'react'
```

Comment:
```suggestion
import type { ReactElement } from 'react'
import { useEffect } from 'react'
```

Reply:
probably better not to pull in the entire entrypoint as a good practice. not a blocker though